ChangeLog

```diff
@@ -40,6 +40,7 @@ ver. 0.9.2 (2014/XX/XXX) - wanna-be-released
    * firewallcmd-* actions: split output into separate lines for grepping (gh-908)
    * Guard unicode encode/decode issues while storing records in the database.
      Fixes "binding parameter error (unsupported type)" (gh-973), thanks to kot for reporting
+   * filter.d/asterisk.conf - Match hacking attempt IP instead of asterisk server IP
```

Comment:
better be more explicit here:  dropped "Sending fake auth rejection" failregex since it incorrectly targets the originating server?  But I would need to better grasp the source/semantic of this msg . 

added catching "hacking attempt detected"